pkgs/by-name/ke/kea/package.nix

```
@@ -24,17 +24,13 @@
 
 stdenv.mkDerivation rec {
   pname = "kea";
-  version = "2.6.2"; # only even minor versions are stable
+  version = "3.0.0"; # only even minor versions are stable
 
   src = fetchurl {
-    url = "https://ftp.isc.org/isc/${pname}/${version}/${pname}-${version}.tar.gz";
-    hash = "sha256-ilC2MQNzS1nDuGGczWdm0t/uPwLjpfnzq8HNVfcPpCQ=";
+    url = "https://ftp.isc.org/isc/${pname}/${version}/${pname}-${version}.tar.xz";
```

https://github.com/NixOS/nixpkgs/issues/310373

```suggestion
    url = "https://ftp.isc.org/isc/kea/${version}/kea-${version}.tar.xz";
```